rosdep/base.yaml

```diff
@@ -3379,6 +3379,146 @@ libpcl-all-dev:
     xenial: [libpcl-dev]
     yakkety: [libpcl-dev]
     zesty: [libpcl-dev]
+libpcl-common:
+  debian: [libpcl-common1.8]
+  ubuntu:
+    artful: [libpcl-common1.8]
+    bionic: [libpcl-common1.8]
+    cosmic: [libpcl-common1.8]
+    disco: [libpcl-common1.8]
+    eoan: [libpcl-common1.8]
+    focal: [libpcl-common1.8]
+    precise: [libpcl-1.7-common1.8]
+    quantal: [libpcl-1.7-common1.8]
+    raring: [libpcl-1.7-common1.8]
+    saucy: [libpcl-1.7-common1.8]
+    trusty: [libpcl-1.7-common1.8]
```

Comment:
@YoshuaNava hmm, I can't seem to even find these fine grained PCL releases on `trusty`.